dali/test/python/test_dali_tf_dataset_graph.py

```diff
@@ -58,13 +65,8 @@ def run_tf_dataset_with_random_input(dev, max_shape, dtype, batch):
     run_tf_dataset_graph(
         dev,
         get_pipeline_desc=external_source_tester(max_shape, dtype, iterator, batch=batch),
-        to_dataset=external_source_converter_with_callback(RandomSampleIterator,
-                                                           max_shape,
-                                                           dtype,
-                                                           0,
-                                                           1e10,
-                                                           min_shape,
-                                                           batch=batch))
+        to_dataset=external_source_converter_with_callback(RandomSampleIterator, max_shape, dtype,
+                                                           0, 1e10, min_shape, batch=batch))
```

Comment:
Why?

Reply:
I believe this enhances readability - this way it's more obvious that these arguments are not to the `run_tf_dataset_graph` function, but the args to the `external_source_converter_with_callback` function.

Reply:
Fair enough.